src/ol/interaction/drawinteraction.js

```diff
@@ -715,6 +715,29 @@ ol.interaction.Draw.prototype.abortDrawing_ = function() {
 
 
 /**
+ * Continue drawing a feature.  This only works on line features, that is
+ * features with a `LINE_STRING` geometry.
+ * @param {!ol.Feature} feature Feature to continue drawing.
```

Comment:
`!ol.Feature` is perfectly valid. I'm just wondering if the compiler can detect that passing `null` is wrong because `getGeometry()` is called in this function.

Side note: the new version of the compiler has a check that prevents the use of the implicit nullable convention. Example:
```
ERR! compile /home/fredj/public_html/ol3/src/ol/map.js:103: ERROR - Name Array in JSDoc is implicitly nullable.
ERR! compile Please add a ! to make it non-nullable, or a ? to make it explicitly nullable.
ERR! compile  * @type {Array.<ol.RendererType>}
```